src/util/NumberContext.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { createContext, useContext } from 'react';
+
+interface NumberContextProps {
+  numberArray: Number[];
+
+}
+
+const NumberContext = createContext<NumberContextProps | undefined>(undefined);
```

Comment:
I don't think this name is very helpful. By calling the context `NumberContext` you're not really telling the user of the component what the number is supposed to be.

Each context can only occur once in an application, so this context can only be used to store a single value. It's not possible to reuse contexts, so giving them very reusable names is unnecessary. If you're intending to use this context to store and share the IDs of selected seats, I would call this context something like `SeatSelectionContext`.